### PR TITLE
Update casper.rst  PR #1459

### PR DESCRIPTION
**Pull request by Macek007**
Doc upadate to PR https://github.com/n1k0/casperjs/pull/1459

### CODE REVIEW

File: docs/modules/casper.rst [hunk 1]
```diff
@@ -1661,6 +1661,60 @@ Options
...
 
+``setFieldValue()``
+-------------------------------------------------------------------------------
+
+**Signature:** ``setFieldValue(Selector selector, Mixed value [,String form [, Object options]])``
+
+.. versionadded:: 1.1-beta5
+
+Sets a value to form field (input) by CSS3, XPath :doc:`selector <../selectors>`:: or by it's ``name`` attribute or ``label`` text.
+For ``name`` or ``label`` selectors are functions aliasies `setFieldValueName()`_ and `setFieldValueLabel()`_ available::
```
Comment (mickaelandrieu) [inline docs/modules/casper.rst]: aliases instead, no ? My english is damn bad :)

File: docs/modules/casper.rst [hunk 2]
```diff
@@ -1661,6 +1661,60 @@ Options
...
+        this.setFieldValue('inputName', 'Input selected by name attribute', null, {selectorType: 'name'});
+        this.setFieldValue('Input Label', 'Input selected by label text', null, {selectorType: 'label'});
+    });
+
+Options
+~~~~~~~
+
+- ``(String) selectorType: name|label|xpath|css3:``
+
+  type of selector, where CSS3 and XPath(object) are autodetected (need not to be set).
```
Comment (mickaelandrieu) [inline docs/modules/casper.rst]: you can remove `(need not to be set)`

File: docs/modules/casper.rst [hunk 3]
```diff
@@ -1712,7 +1766,7 @@ Alternatively::
     casper.then(function() {
         this.echo("I'm loaded.");
     });
-
+
```
Comment (mickaelandrieu) [inline docs/modules/casper.rst]: remove the extra tabulation please :)

**Comment by kensoh** [comment]
Archiving PR using **ARCHIVE** label as CasperJS, like PhantomJS, is not actively maintained for now. These PRs are from 1-5 years ago but there just hasn't been maintainers to continue the work.

**Pull request closed without merging**